composeApp/src/commonMain/kotlin/world/respect/app/viewmodel/apps/detail/AppsDetailScreenViewModel.kt

```diff
@@ -0,0 +1,126 @@
+package world.respect.app.viewmodel.apps.detail
+
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import org.jetbrains.compose.resources.getString
+import world.respect.app.model.appsdetail.AppsDetailModel
+import world.respect.app.viewmodel.RespectViewModel
+import respect.composeapp.generated.resources.Res
+import respect.composeapp.generated.resources.apps_detail
+import world.respect.app.model.appsdetail.Images
+import world.respect.domain.opds.model.LangMapStringValue
+import world.respect.domain.opds.model.OpdsPublication
+import world.respect.domain.opds.model.ReadiumContributorStringValue
+import world.respect.domain.opds.model.ReadiumLink
+import world.respect.domain.opds.model.ReadiumMetadata
+import world.respect.domain.opds.model.ReadiumSubjectStringValue
+
+
+data class AppsDetailUiState(
+    val appsDetailData: AppsDetailModel? = null
+)
+
+class AppsDetailScreenViewModel : RespectViewModel() {
```

Comment:
Not following the pattern: view model class should not have 'Screen' in its name

Reply:
Mostly resolved; however Clazz and Assignment placeholders need to follow the pattern too